hoot-core/src/main/cpp/hoot/core/criterion/ElementInIdListCriterion.h

```diff
@@ -22,39 +22,43 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016, 2017, 2018 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2018 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Is this an actual rename of the file or is GitHub just messed up?  If you deleted `IsNodeFilter.h` and created `ElementInIdListCriterion` (seem to be unrelated) then this copyright is ok.  That is what I suspect happened here.  Is that true?

Reply:
Yeah, I meant to comment that one here on github.  IsNodeFilter was deleted.  Actually, ElementInIdListCriterion is new file that is a rework of IntersectionCriterion, which was poorly named...didn't really have much to do with intersections.  So, the 2018 copyright is ok for this new file.